lib/hubspot/company.rb

```diff
@@ -165,6 +165,18 @@ def add_contact!(company_vid, contact_vid)
                                      },
                                      body: nil)
       end
+
+      # Updates the properties of a company
+      # {http://developers.hubspot.com/docs/methods/companies/update_company}
+      # @param vid [Integer] hubspot company vid
+      # @param params [Hash] hash of properties to update
+      # @return [Hubspot::Company] Company record
+      def update!(vid, params)
+        params.stringify_keys!
+        query = {"properties" => Hubspot::Utils.hash_to_properties(params, key_name: "name")}
```

Comment:
Interesting, I've only looked at a handful of examples but it seems the HubSpot v1 endpoints use "property" and the v2 endpoints use "name"